test/parallel/test-process-load-env-file.js

```diff
@@ -45,13 +45,13 @@ describe('process.loadEnvFile()', () => {
   it('should throw when file does not exist', async () => {
     assert.throws(() => {
       process.loadEnvFile(missingEnvFile);
-    }, { code: 'ENOENT' });
+    }, /ENOENT: no such file or directory, open '.*non-existent-file\.env'/);
```

Comment:
We shouldn't check for error messages. The proper way of testing this would be:

```suggestion
    }, { code: 'ENOENT', errno: -2, syscall: 'open' });
```

Reply:
Done ✅